sessions/session-b.md

```diff
@@ -0,0 +1,36 @@
+# Session B: Creating low-power sensor networks using LoRa
+
+## Description
+
+The last few year have been an exciting time for the DIY electronics community. There has been an explosion of sensors, actuators, and micro-controllers to tinker with. Additionally, with access to inexpensive hardware like the Photon, Raspberry Pi, and ESP8266 it’s easier than ever to embed web-services directly in to our projects and this connectivty enables new applications. However, most of these projects connect over WiFi, which isn't well suited for low-power/low-bandwidth applications. WiFi usually limits our projects to indoors or tethered to our smartphones. These mobility and power limitations impede projects like distributed citizen science, connected guerrilla art, and remote sensing. LoRa is an inexpensive low-power/low-bandwidth radio technology that is one great solution to this problem. In urban settings LoRa radios can often provide low bandwidth connectivity of up to 1-2 kms and can be reasonably powered with AA batteries.
+
+In this session I will give a brief overview of LPWAN's (low-power wide area networks) and the technology behind LoRa. There is also a mini wokshop component to this session where we will set up infrastructure for an LPWAN to monitor sensors connect through LoRa radios.
+
+## Session Objective
+
+* Introduce LPWANs and LoRa
+* Open discussion about IoT and issues that arise through pervasive connectivity and a path forward
+* Hands-on demo of LoRa radios
+
+## Length
+
+50 mins
+
+## Materials
+
+* I have a few Adafruit LoRa feather boards (include LoRa radio + Arm M0 microcontroller board) for loan or purchase (each is about CAD $50)
+* Participants are encouraged to bring any arduinos or microcontroller projects they would like to add LoRa connectivity to
```

Comment:
Do participants need a computer?
What materials do you as a presenter need that we (I) should make sure to have?